yal/tests/test_askaquestion.py

```diff
@@ -413,51 +427,76 @@ async def test_state_display_content_question_not_answered(
     tester.setup_state("state_display_content")
     tester.setup_answer("state_display_results", "FAQ #1 Title")
 
-    await tester.user_input("No")
+    await tester.user_input("Nope...")
 
-    tester.assert_state("state_display_results")
+    tester.assert_state("state_no_question_not_answered")
 
-    assert len(rapidpro_mock.tstate.requests) == 1
-    request = rapidpro_mock.tstate.requests[0]
-    assert json.loads(request.body.decode("utf-8")) == {
-        "fields": {"aaq_timeout_type": ""},
-    }
+    message = "\n".join(
+        [
+            "*I'm sorry I couldn't find what you were looking for this time.* ",
+            "",
+            "Please tell me what you're looking for again. "
+            "I'll try make sure I have the right information "
+            "for you next time.",
+            "",
+            "_Just type and send your question again, now._" "",
+            "-----",
+            "*Or reply:*",
+            BACK_TO_MAIN,
+            GET_HELP,
+        ]
+    )
 
-    assert len(aaq_mock.tstate.requests) == 1
-    request = aaq_mock.tstate.requests[0]
-    assert json.loads(request.body.decode("utf-8")) == {
-        "feedback": {"feedback_type": "negative", "faq_id": "1"},
-        "feedback_secret_key": "feedback-secret-key",
-        "inbound_id": "inbound-id",
-    }
```

Comment:
These lines need to still be here because we still need to do this for negative responses (this makes sure we're updating the model with the negative response feedback)